MANIFEST.in

```diff
@@ -1,19 +1,9 @@
-include dom0/*.sls
-include dom0/*.top
-include dom0/*.j2
-include dom0/*.yml
-include dom0/*.conf
-include dom0/remove-tags.py
-include dom0/securedrop-handle-upgrade
+include securedrop_salt/*
```

Comment:
This includes the apt-test key and sources file. We probably should be excluding these and adding them in via the dev setup script when `make dev` or `make staging` is run instead. They shouldn't be shipped in the prod RPM.